From a39e95e9317fdad4ebaf15cf6b4ef84ce94e86ef Mon Sep 17 00:00:00 2001
From: Subash Abhinov Kasiviswanathan <subashab@codeaurora.org>
Date: Fri, 4 Sep 2015 18:33:39 -0600
Subject: [PATCH 2/2] net: include: rmnet_data: Define the MAPv4 data format

Add an entry corresponding to the MAPv4 ingress and egress data format
for checksum offload. Also update the copyright.

Change-Id: Id2368d621f48ebf510371acf1502efb8bead65d2
Signed-off-by: Subash Abhinov Kasiviswanathan <subashab@codeaurora.org>
Signed-off-by: sub77 <sub77@ymail.com>
---
 include/linux/rmnet_data.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/include/linux/rmnet_data.h b/include/linux/rmnet_data.h
index e0ca5116e7d..92c04b3ad6c 100644
--- a/include/linux/rmnet_data.h
+++ b/include/linux/rmnet_data.h
@@ -9,6 +9,7 @@
 #define RMNET_EGRESS_FORMAT_AGGREGATION         (1<<2)
 #define RMNET_EGRESS_FORMAT_MUXING              (1<<3)
 #define RMNET_EGRESS_FORMAT_MAP_CKSUMV3         (1<<4)
+#define RMNET_EGRESS_FORMAT_MAP_CKSUMV4         (1<<5)
 
 #define RMNET_INGRESS_FIX_ETHERNET              (1<<0)
 #define RMNET_INGRESS_FORMAT_MAP                (1<<1)
@@ -16,6 +17,7 @@
 #define RMNET_INGRESS_FORMAT_DEMUXING           (1<<3)
 #define RMNET_INGRESS_FORMAT_MAP_COMMANDS       (1<<4)
 #define RMNET_INGRESS_FORMAT_MAP_CKSUMV3        (1<<5)
+#define RMNET_INGRESS_FORMAT_MAP_CKSUMV4        (1<<6)
 
 /* ***************** Netlink API ******************************************** */
 #define RMNET_NETLINK_PROTO 31
-- 
2.11.0

